lib/rust/ensogl/component/list-view/src/entry.rs

```diff
@@ -240,9 +240,14 @@ impl display::Object for GlyphHighlightedLabel {
 /// for models of entries when they're about to be displayed. So setting the select content is
 /// essentially providing an implementor of this trait.
 pub trait ModelProvider<E>: Debug {
-    /// Number of all entries.
+    /// Number of entries after filtering.
     fn entry_count(&self) -> usize;
 
+    /// Number of all entries before any filtering.
+    fn original_entry_count(&self) -> usize {
```

Comment:
The context of these docs is unknown. The ModelProvider is designed primarily for ListView component, which does not have anything related to filtering.

I think it's better to add `original_entry_count` field to `column_grid::LabeledAnyModelProvider` - there we are in the context of component browser, and we should know how filtering works

Reply:
My concern was that LabeledAnyModelProvider contains the reference to the list of entries, and this list can be changed silently in some other place without updating the `original_entry_count` field. Done.